bundles/org.openhab.binding.pulseaudio/README.md

```diff
@@ -59,6 +59,14 @@ Sink things can register themselves as audio sink in openHAB. MP3 and WAV files
 Use the appropriate parameter in the sink thing to activate this possibility (activateSimpleProtocolSink).
 This requires the module **module-simple-protocol-tcp** to be present on the server which runs your openHAB instance. The binding will try to command (if not discovered first) the load of this module on the pulseaudio server.
 
+
+## Audio source
+
+Source things can register themselves as audio source in openHAB.
+WAV input format, rate and channels can be configured on the thing configuration. (defaults to pcm_signed,16000,1)
+Use the appropriate parameter in the source thing to activate this possibility (activateSimpleProtocolSource).
+This requires the module **module-simple-protocol-tcp** to be present on the target pulseaudio server. The binding will load this module on the pulseaudio server.
```

Comment:
What is the "target pulseaudio server". Is it "server which runs your openHAB instance" like for the sink thing ? Or is it the remote server with the audio source ?

Reply:
It could be both, it's a server with a running Pulseaudio instance.
What could be a better expression for it?

Reply:
Ok, I understand, it could be a remote server or the same server as the OH server. In the case of a local source, it makes this an alternative to javasound if I correctly understand.
So you have nothing to change.